draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -160,6 +160,15 @@ specifies the use of the ML-DSA in Public Key Infrastructure X.509 (PKIX)
 certificates and Certificate Revocation Lists (CRLs) at three security
 levels: ML-DSA-44, ML-DSA-65, and ML-DSA-87.
 
+{{FIPS204}} defines two variants of ML-DSA: a pure and a prehash variant.
+Only the former is specified in this document.
+The pure variant of ML-DSA supports the typical prehash flow:
+one cryptographic module can compute the hash *mu*
+on line 6 of algorithm 7 of {{FIPS204}} and pass it to a second module
+to finish the signature. The first module only needs access to the full
+message and the public key, whereas the second module only needs access
+to hash *mu* and the private key.
```

Comment:
Nit. Maybe use function instead of module. I would refrain from using the word module. We don't need FIPS nomenclature in this doc. We just explain this can be done. If both modules will be FIPS or one, or none, we don't really care in this spec.